docker/test

```diff
@@ -0,0 +1,52 @@
+#! /usr/bin/env bash
+set -eux -o pipefail
+
+pushd "$(git -C "$(dirname "$(readlink -f "${0}")")" rev-parse --show-toplevel)/docker/vast"
+
+# We set a unique project name during testing so we can check at the end whether
+# all containers we spun up during testing exited cleanly. Docker Compose uses
+# this as a prefix for the service, network, and volume names unless we set a
+# custom 'container_name', which we do not.
+export COMPOSE_PROJECT_NAME="vast-test-${RANDOM}"
+
+# Bring up the Docker Compose stack.
+docker compose down --volumes
+docker compose up --no-build --detach
```

Comment:
- `no-build` should not be necessary here as you need the overlay `-f docker-compose.build.yaml` to setup the build, right?
- I am confused now, don't we need to use the `ghcr.io/tenzir/vast:${cache_to_ref}` cache here?

Reply:
We talked about this earlier in a meeting, and agreed that we actually need two modes for the test script: One where we build what is locally available, and one where we pull, with the latter setting `VAST_CONTAINER_{REF,REGISTRY}` automatically depending on the local branch.